content/en/docs/client-options.md

```diff
@@ -60,6 +60,7 @@ These clients are compatible with our [staging endpoint for ACME v2](https://com
 - [Nginx ACME V1/V2 client](https://github.com/kshcherban/acme-nginx)
 - [WinCertes Windows ACMEv2 client](https://github.com/aloopkin/WinCertes)
 - [ACME-PS](https://github.com/PKISharp/ACMESharpCore-PowerShell)
+- [FreeSSL.tech Auto](https://freessl.tech) (It works with Live ACME V2 API. Auto-installs SSL cert in shared hosting cPanel. No root access required.)
```

Comment:
Can you please remove the explanatory text in the trailing parentheses? It's too verbose and I'd prefer we don't try to enumerate features in the client list. Thanks!